src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -21,8 +21,6 @@
     is_windows, is_wsl
 from azure.cli.core.cloud import get_active_cloud, set_cloud_subscription
 
-from .adal_authentication import MSIAuthenticationWrapper
```

Comment:
@jsntcy, relative import it is not recommended according to [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports):

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages)...